src/Elmish.WPF/ViewModelArgs.fs

```diff
@@ -27,13 +27,11 @@ module internal LoggingViewModelArgs =
       nameChain = "" }
 
 
-type internal ViewModelArgs<'model, 'msg> =
-  { initialModel: 'model
-    dispatch: 'msg -> unit
-    loggingArgs: LoggingViewModelArgs }
+type ViewModelArgs<'model, 'msg> =
+  internal { initialModel: 'model; dispatch: 'msg -> unit; loggingArgs: LoggingViewModelArgs }
```

Comment:
Please keep each field on it's own line.  In particular, this will keep the diff smaller.